src/content/docs/style-guide/writing-strategies/nav-best-practices.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Docs nav best practices
+metaDescription: "Everything I know about information architecture I learned in library science school."
+---
+
+What is the purpose of a navigation sidebar menu? This is worth considering, because this reader experience is pretty ubiquitous across doc sites, and things that are everywhere tend not to be considered much.
```

Comment:
This sentence feels clunky to me although totally true and totally accurate.  "This is worth considering, because this reader experience is pretty ubiquitous across doc sites, and things that are everywhere tend not to be considered much." 

I have an impulse to rewrite it to say something like "The navigation experience is ubiquitous for readers across most doc sites but because this experience is so common to our every day lives, we often don't consider its importance."

Reply:
Actually, never mind. The way you wrote it is fine. Upon revisiting this, I see that I literally just structured my sentence the same way.

Reply:
LOL ok, I'll think about it some more.